cmd/controller/main.go

```diff
@@ -43,6 +43,13 @@ const (
 )
 
 func main() {
+	if val, ok := os.LookupEnv("THREADS_PER_CONTROLLER"); ok {
+		threadsPerController, err := strconv.Atoi(val)
+		if err != nil {
+			log.Fatalf("failed to parse value %q of THREADS_PER_CONTROLLER: %v\n", val, err)
+		}
+		controller.DefaultThreadsPerController = threadsPerController
```

Comment:
This means that the environment variable overrides the default, and the command line arg takes precedence, right? Is this consistent with the behaviour of the other two environment variables?